recipes/pixmappy/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "pixmappy" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/gbernstein/pixmappy/archive/v{{ version }}.tar.gz
+  sha256: 9ef1ddeaea58ca50cbf7226a2c68ac183726ae3e97028f9011849dd086f7c48d
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - numpy
+    - future
+    - astropy
+    - scipy
+    - pyyaml
+    - lsstdesc.coord
+
+test:
+  imports:
+    - pixmappy
+
+about:
+  home: https://github.com/gbernstein/pixmappy
+  license: BSD-2-Clause
+  license_family: BSD
+  license_file: LICENSE
```

Comment:
I think you need to include this as well: https://github.com/gbernstein/pixmappy/blob/master/Piff_LICENSE

Reply:
Good catch!

Reply:
done!